docs/configuration.md

```diff
@@ -364,3 +365,15 @@ prometheus:
 |-----------|----------|------------------|-----------------------------------------|
 | `enabled` | `bool`   | `false`          | Flag to enable the service.             |
 | `address` | `string` | `localhost:8086` | Address that service listener binds to. |
+
+# `neofs` section
+
+Contains parameters of requests to NeoFS.
+
+```yaml
+neofs:
+  set_copies_number: 0
+```
+| Parameter           | Type   | Default value | Description                                                        |
+|---------------------|--------|---------------|--------------------------------------------------------------------|
+| `set_copies_number` | uint32 | 0             | Number of object copies that is enough to consider put successful. |
```

Comment:
Please mention that `0` is some default behavior rather than `zero copies is enough to consider put successful`  
```suggestion
| `set_copies_number` | uint32 | `0`             | Number of object copies that is enough to consider put successful. |
```